addons/survey/report/survey_templates.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <template id="certification_report_view_general">
+            <!-- Style classes to be applyed to '#o_survey_certification': [no class](purple), gold, blue -->
```

Comment:
While you're at it, there is a small typo "applyed" :eyes:

Reply:
@awa-odoo everything looks good now :+1: